canvasapi/paginated_list.py

```diff
@@ -5,6 +5,7 @@ class PaginatedList(object):
     """
     Abstracts `pagination of Canvas API \
     <https://canvas.instructure.com/doc/api/file.pagination.html>`_.
+    Returns a complete list of all results of a specified type.
```

Comment:
This is a bit misleading since PaginatedList doesn't actually return a list, let alone a complete one. It kinda grabs what it needs while it goes.

Maybe mention that it's an iterator, or something along those lines?

Reply:
I went back and forth on that last night. I ended up just taking it out because just having `__init__` documented will help.